bigtable/google/cloud/bigtable/row.py

```diff
@@ -907,15 +923,13 @@ def commit(self):
             )
 
         data_client = self._table._instance._client.table_data_client
-        row_response = data_client.read_modify_write_row(
+        data_client.read_modify_write_row(
             table_name=self._table.name, row_key=self._row_key, rules=self._rule_pb_list
         )
 
         # Reset modifications after commit-ing request.
         self.clear()
-
-        # NOTE: We expect row_response.key == self._row_key but don't check.
-        return _parse_rmw_row_response(row_response)
+        return self.row_content
```

Comment:
This is also bad.  This causes an extra lookup.  `row_response` is already equivalent to `row_content` without requiring a lookup

Reply:
As I understood, the request was to return the whole row, while `row_content` contains only modified cells
https://github.com/googleapis/google-cloud-python/blob/fa6e4f5784f0063ff832159e561aa89d45497dea/bigtable/google/cloud/bigtable/row.py#L939